apps/fluent-tester/src/FluentTester/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Platform, View } from 'react-native';
+import { FAB, Text } from '@fluentui/react-native';
+import { commonTestStyles, stackStyle } from '../Common/styles';
+
+export const ShadowButtonTestSection: React.FunctionComponent = () => {
+  if (Platform.OS === 'ios') {
+    const CustomFABNoShadow = FAB.customize({ shadowDepth: undefined });
```

Comment:
customized components should be created outside of render functions, since the function ends up creating a new object when run.

Reply:
Got it, thanks for catching that and explaining why